src/main/java/net/dv8tion/jda/internal/utils/PermissionUtil.java

```diff
@@ -314,7 +314,8 @@ public static long getEffectivePermission(Member member)
             if (isApplied(permission, Permission.ADMINISTRATOR.getRawValue()))
                 return Permission.ALL_PERMISSIONS;
         }
-
+        if (member.isTimedOut())
+            return Permission.getRaw(Permission.VIEW_CHANNEL, Permission.MESSAGE_HISTORY);
```

Comment:
This does not work when `permissions = 0`

Reply:
oh, I get it now. thanks